### PR TITLE
UI add Method get

### PR DESCRIPTION
**Pull request by Cxristian**
get Duration

### CODE REVIEW

File: src/main/java/com/jala/view/CustomTitleBorder.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import javax.swing.border.LineBorder;
+import javax.swing.border.TitledBorder;
+import javax.swing.plaf.ColorUIResource;
+import java.awt.Color;
+import java.awt.Font;
+
+public class CustomTitleBorder extends TitledBorder {
+
+    public CustomTitleBorder(String title) {
+
```
Comment (AKatrin) [inline src/main/java/com/jala/view/CustomTitleBorder.java:37]: remove empty line please

File: src/main/java/com/jala/view/CustomTitleBorder.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import javax.swing.BorderFactory;
+
+import javax.swing.border.CompoundBorder;
+import javax.swing.border.EmptyBorder;
+import javax.swing.border.LineBorder;
+import javax.swing.border.TitledBorder;
+import javax.swing.plaf.ColorUIResource;
+import java.awt.Color;
+import java.awt.Font;
+
```
Comment (AKatrin) [inline src/main/java/com/jala/view/CustomTitleBorder.java:22]: Add documentation

File: src/main/java/com/jala/view/JPanelSearchGral.java [hunk 3]
```diff
@@ -115,11 +114,9 @@ private void initComponent() {
      */
     private JPanel pnlCriteria() {
         JPanel pnlPath = new JPanel(new FlowLayout(FlowLayout.LEFT));
-        TitledBorder titleBorder = BorderFactory.createTitledBorder(null, "Search");
-        titleBorder.setTitlePosition(TitledBorder.DEFAULT_POSITION);
-        titleBorder.setTitleFont(new Font("Arial", 1, 10));
-
+        TitledBorder titleBorder = new CustomTitleBorder("Search: ");
         pnlPath.setBorder(titleBorder);
+        pnlPath.setBackground(new Color(0,0,0,65));
```
Comment (AKatrin) [inline src/main/java/com/jala/view/JPanelSearchGral.java]: please do format code

File: src/main/java/com/jala/view/JPanelSearchVideo.java [hunk 4]
```diff
@@ -205,6 +205,21 @@ private void constraintComponent(Component component, int posX, int posY, int al
         add(component, constraints);
     }
 
+    public JRadioButton getRadioStereo() {
```
Comment (AKatrin) [inline src/main/java/com/jala/view/JPanelSearchVideo.java:211]: Add docummentation

**Review by raulchoque-eng — APPROVED** [review]

**Pull request merged**